network-container/README.md

```diff
@@ -10,4 +10,24 @@ Next steps:
    You can also do this manually with the following command: `cfn-cli generate`
 3. Implement your resource handlers by adding code to provision your resources in your resource handler's methods.
 
-Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
\ No newline at end of file
+Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
+
+## Description
+This resource allows you to create, read, update and delete a network container
+
+## Attributes
+`Id` : Unique identifier of the Network Peer.<br>
+`Provisioned` : Flag that indicates if the project has clusters deployed in the Network Peering container.<br>
+`VpcId` : Unique identifier of the peer VPC.<br>
+
+## Parameters
+`ProjectId` *(required)* : The unique identifier of the project.<br>
+`RegionName` *(optional)* : Name of region<br>
+`ProviderName` *(optional)* : The name of the provider.<br>
```

Comment:
Should we just put AWS here to reduce confusion?

Reply:
Sure but did you mean with parameter or adding like Default is `AWS`? @themantissa